demos/yelp_demo/README.md

```diff
@@ -209,43 +185,34 @@ Create the serving and UI components:
 ks apply gke -c serving -c ui
 ```
 
-Forward the port for viewing from a local browser:
+Connect to the UI by forwarding a port to one of the ambassador
+pods:
+
 ```
-UI_POD=$(kubectl get po -l app=kubeflow-demo-ui | \
-  grep kubeflow-demo-ui | \
+AMBASSADOR_POD=$(kubectl get po -l service=ambassador | \
+  grep ambassador | \
   head -n 1 | \
   cut -d " " -f 1 \
 )
-kubectl port-forward ${UI_POD} 8080:80
+kubectl port-forward ${AMBASSADOR_POD} 8080:80
```

Comment:
Why port-foward ambassador? Are we using it like ingress?

Reply:
Isn't that the point of the reverse proxy - to port-forward once and access various components via pathname?

Reply:
I guess for the demo purpose use ambassador as a reverse proxy should be fine.